src/Symfony/Component/Translation/Bridge/Phrase/README.md

```diff
@@ -0,0 +1,99 @@
+# phrase-translation-provider
+
+symfony phrase translation provider bridge
+
+## dsn example
```

Comment:
```suggestion
## DSN Example
```

Reply:
yes, i've got a thing against capitals in typed language: i'll "fix" the readme 😃